scene/gui/tree.cpp

```diff
@@ -1404,6 +1442,21 @@ String TreeItem::get_tooltip_text(int p_column) const {
 	return cells[p_column].tooltip;
 }
 
+void TreeItem::set_tooltip_visible(int p_column, bool p_visible) {
+	ERR_FAIL_INDEX(p_column, cells.size());
+
+	if (cells[p_column].tooltip_visible == p_visible) {
+		return;
+	}
```

Comment:
Setting tooltip visibility just flips a boolean field, so this check is unnecessary.